frontend/src/app/services/game.ts

```diff
@@ -101,34 +103,53 @@ export class Game {
     });
   }
 
-  private calcMitigationEffect(mitigations: Mitigations, date: string): MitigationEffect {
-    let mult = 1.0;
-    let cost = 0;
-    let stabilityCost = 0;
+  addEventMitigation(mitigation: EventMitigation) {
```

Comment:
This should be called when the player clicks one of the event mitigation options